valhalla/baldr/double_bucket_queue.h

```diff
@@ -106,7 +107,10 @@ class DoubleBucketQueue {
    * @param  cost  Cost.
    * @return Returns the bucket that the cost lies within.
    */
-  bucket_t& get_bucket(const float cost) {
+  bucket_t& get_bucket(float cost) {
+    //shift the cost to the middle of the bucket so as to avoid precision issues
+    //since minimum bucketsize is 1 this shouldnt be a problem
+    cost = std::floor(cost) + (bucketsize_ *.5f);
```

Comment:
bucketsize_ * 0.5f is constant - should it be a data member to avoid multiplication?